Bowling.java

```diff
@@ -74,8 +74,13 @@ public String printFrame() {
 
 class Scoreboard {
 
+	private static final int strikePoints = 10;
+	private static final int sparePoints = 10;
 	private List<Frame> allFrames;
 	private Integer totalPoints;
+	private Optional<Frame> secondToLastFrame;
+	private Optional<Frame> thirdToLastFrame;
+	private Frame currentFrame;
```

Comment:
I see why these were added, but they make me nervous.  There value is set as a side effect of the `updateTotalPoints` method.  So I would totally expect `totalPoints` to have an updated value that I could use.  These, however, will change at some point in the processing and I'm not sure it will be clear if our code gets more complex.

I might be being too paranoid, but these values are only used by two very specific methods and if we ever wanted to move those methods we'd have to be careful to move the code setting these values as well.